reviewing.md

```diff
@@ -49,6 +49,16 @@ the `name` and `website_url` are listed.
 fashion, verify that the `name` is listed in "Participant Kubernetes
 Combinations" section of the vendor's Participation Form.
 
-When a submission meets all policy requirements, label the pull request
-`participation-form-signed-yes` to indicate that the policy review is complete.
+When a submission meets all policy requirements, reply to the pull request
+with `Signed Participation Form Received` to indicate that the policy review is complete.
```

Comment:
If the submission doesn't meet all policy requirements, reply with a message indicating "Signed participation form needed", "Files missing from PR", "Membership in CNCF or confirmation of non-profit status needed", etc.